docker-compose.prod.yml

```diff
@@ -131,13 +130,11 @@ services:
     ports:
       - "8081:8081"
     volumes:
-      - ./dbsync-central/application.prod.properties:/home/eip/application.properties
```

Comment:
Where will the application.properties file come from?

Reply:
Now I've added all the stuff for dbsync on [release_stuff folder](https://github.com/FriendsInGlobalHealth/centralization-docker-setup/tree/EC-113/dbsync-central/release_stuff)